pom.xml

```diff
@@ -20,8 +20,18 @@
         <spring-boot-dependencies.version>3.1.5</spring-boot-dependencies.version>
         <log4j.version>2.17.2</log4j.version>
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+        <sonar-maven-plugin.version>3.9.1.2184</sonar-maven-plugin.version>
 	</properties>
 
+    <profiles>
+        <profile>
+            <id>sonar-pr-analysis</id>
+            <properties>
+                <sonar.pullrequest.base>main</sonar.pullrequest.base>
```

Comment:
I think there might be another couple of properties needed here. By default sonar I think attempts to use localhost so the concourse tasks reset the login properties in the pom based on values from the pipeline:

See profile and properties here - https://github.com/companieshouse/parent-pom/blob/master/pom.xml

Reply:
Have switched to companies-house-parent now so can you review again?